php/5.6/Dockerfile

```diff
@@ -0,0 +1,96 @@
+FROM php:5.6-fpm
+
+LABEL maintainer="Abhijit Rakas <abhijitrakas@gmail.com>"
+LABEL org.label-schema.schema-version="1.0.0-rc1"
+LABEL org.label-schema.vendor="EasyEngine"
+LABEL org.label-schema.name="php"
```

Comment:
We need another label to differentiate between the PHP7 and PHP5.6 containers. We can use `org.label-schema.version` as mentioned here - http://label-schema.org/rc1/

Reply:
@abhijitrakas Similarly for PHP7 image also, we need to add the label.